com.unity.multiplayer.mlapi/Tests/Editor/SnapshotRttTests.cs

```diff
@@ -0,0 +1,33 @@
+using MLAPI;
+using NUnit.Framework;
+
+namespace MLAPI.EditorTests
+{
+    public class SnapshotRttTests
+    {
+        [Test]
+        public void TestRtt()
+        {
+            var snapshot = new SnapshotSystem();
+            var client1 = snapshot.Rtt(0);
+
+            client1.NotifySend(0, 0.0);
+            client1.NotifySend(1, 10.0);
+            client1.NotifySend(2, 20.0);
+            client1.NotifySend(3, 30.0);
+
+            client1.NotifyAck(1, 15.0);
+            client1.NotifyAck(3, 40.0);
+
+            double epsilon = 0.0001;
```

Comment:
constant `Double.Epsilon` is already a thing in C#/.Net — would it succeed with that instead of arbitrary `0.0001`?

Reply:
No, it would not. 
`public const double Double.Epsilon = 4.94065645841247E-324;` is the smallest possible float value. Adding it to anything of significant magnitude will result in no change at all. Here, we want some tolerance because floating-point computations are not exact. I'll agree that 1e-4 might be on the larger side of epsilons, but still, I feel that's a reasonable range for precision of an average in these ranges (5.0-10.0) .

Reply:
I used a similar pattern for time tests and agree with Jeff here.